src/controllers/users.js

```diff
@@ -38,4 +38,43 @@ export default class UsersController {
       return errorResponse(res, status.error, messages.signUp.error);
     }
   }
+
+  /**
+   * @method signInUser
+   * @description Method for user sign in
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @returns {object} response body object
+   */
+  static async signInUser(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await models.Users.findOne({ where: { email } });
+      if (!user) {
+        return errorResponse(res, status.notfound, messages.signIn.invalid);
```

Comment:
For the same security as mention by @Oliver-ke, the **status code** for the error should be `unathorized`